.github/workflows/release-candidate.yml

```diff
@@ -0,0 +1,75 @@
+name: Create a release candidate bundle that is uploaded to Google Play internal testing track
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'rc*'
```

Comment:
The tag is also used to determine the marketing version name, by removing the first char. (eg tag `v1.2.3` will create a app-versionname of `1.2.3-git-sha`. Adjust the logic for creating the version-name, of change the naming scheme for release candidates.